v1.1.0_release_notes.md

```diff
@@ -0,0 +1,59 @@
+## Breaking Changes.
+dcrpool v1.1.0 introduces breaking changes to the configuration, existing configurations will have to be updated accordingly. The breaking 
+changes made are:
+ 1. `--backuppass` has been renamed to `--adminpass`.
+
+ 2.  `--lastnperiod` and `--maxgentime` are now of type `time.Duration`.
+
+ 3. Added `--walletaccount` config option.
+
+ 4. Deprecated `--minpayment` & `--maxtxfeereserve`.
+
+## Bug Fixes.
+ The notable bug fixes in this release include:
+   1. Data access issues related to using raw db values outside 
+   of their associated db txs have been resolved.
+
+   2. Race conditions associated with the listing websocket clients and 
+   updating the gui have been resolved.
+
+   3. Panics associated with the cpu miner in the testing harness have been resolved.
+
+   4. A case where a client stalling on work does not receive timestamp-rolled work has been resolved.
+
+   5. Panics associated with logging when the logger has not been initialized have been resolved.
+
+   6. A PPS bug where shares generated outside the range of the confirmed work are used in generating payments has been fixed.
+
+## Improvements.
+The notable infrastructue and interface improvements made include:
+ 1. UI/UX has been refreshed with the aim of simplifying usage patterns.
+
+ 3. Reworked payments processing by having payments track the coinbases being sourced from as well as ensuring payouts are made immediately the associated coinbase is mature and spendable. For this reason, payout transactions do not return change.
+
+ 4. Test coverage for pool components have been improved to ensure the 
+ components do what they are expected to under various conditions and also 
+ ensure added features do not break expected behaviour.
+
+ 5. Support for the Obelisk DCR1 has been added. dcrpool currently supports all publicly available ASICs for decred at the moment.
+
+ 6. `--maxconnectionsperhost` configuration option can now be used to limit the number of connections the pool will accept from a single host, the default limit is a 100 clients. 
+
+ 7. Work is now delivered via dcrd's notifywork which sends work notifications which is more efficient than polling the `getwork` rpc.
+ 
+ 8. Timestamp-rolled work support has been added to distribute updated work 
+ miners stall on work submissions.
+
+ 9. GUI caching and cache update signalling have been added.
```

Comment:
Maybe explain that this reduces load on the host because each page load is getting data from memory rather than the db.